diff --git a/node_modules/react-native-screens/android/CMakeLists.txt b/node_modules/react-native-screens/android/CMakeLists.txt
index 86ec079..b359251 100644
--- a/node_modules/react-native-screens/android/CMakeLists.txt
+++ b/node_modules/react-native-screens/android/CMakeLists.txt
@@ -47,6 +47,8 @@ if(${RNS_NEW_ARCH_ENABLED})
             ReactAndroid::jsi
             fbjni::fbjni
             android
+            log
+            c++_shared
         )
     else()
         target_link_libraries(
